### ISSUE TITLE
Include update/upgrade guidelines in Readme

### ISSUE THREAD

**Issue by HelterBelter** [crowdsecurity/opnsense-plugin-crowdsec#14]
I just saw that v0.0.4 was released and tried to install it.

A fresh install went through fine,

but trying to install os-crowdsec-0.0.4.txz over v0.0.3 failed with

`# pkg add os-crowdsec-0.0.4.txz 
Installing os-crowdsec-0.0.4...
the most recent version of os-crowdsec-0.0.3 is already installed
`

**Comment by nathan-dan** [comment]
I had the same situation occur, with the same error message.  As I'm not as familiar with FreeBSD as I am with Linux, a quick search showed that changing 'add' to 'upgrade' is required for a package update.
`# pkg upgrade os-crowdsec-0.0.4.txz`

After restarting CrowdSec, all appears ok, except for the red cross under **Validated?**  I did get a success message (_machine '--MACHINE-NUMBER--' validated successfully_) in the console, but the GUI still has a red cross.  The command that I used was -
`# cscli machines validate --MACHINE-NUMBER--`
